modules/aurora-mysql/variables.tf

```diff
@@ -159,8 +153,33 @@ variable "publicly_accessible" {
   default = false
 }
 
-variable "eks_component_name" {
+variable "eks_component_names" {
+  type        = set(string)
+  description = "The names of the eks components"
+  default     = ["eks/cluster"]
+}
+
+variable "replication_source_identifier" {
   type        = string
-  description = "The name of the eks component"
-  default     = "eks/eks"
+  description = "ARN of a source DB cluster or DB instance if this DB cluster is to be created as a Read Replica. If this value is empty and replication is enabled, remote state will attempt to find a matching cluster in the Primary DB Cluster's region"
+  default     = ""
 }
+
+variable "replication_enabled" {
+  type        = bool
+  description = "Whether or not to create this DB cluster as a Read Replica."
```

Comment:
This is confusing. I would expect I would need to set `replication_enabled` on the primary cluster in order to create a read replica. I think if this is truly a flag on whether the component is itself a read replica or not, it should be called `is_read_replica` or something like that. We also need to be able to reconfigure these components to promote the read replica to primary and vice versa for disaster recovery, so we might want to call this `is_primary`. @aknysh @osterman What do you think?

Reply:
fair enough, I will update the variable name. But Replication should be enabled by the replica component, since it's deployed independently from the primary cluster, with a separate task/ticket/stack, and in a different region. The replica can be destroyed without worrying about affecting the primary cluster

Reply:
this is for cross-region replication specifically. We need to create / update docs for the differences between global, single regional, cross regional

Reply:
updated variable name and related descriptions